nexus-sdk/src/main/java/io/nexusrpc/OperationFailure.java

```diff
@@ -0,0 +1,120 @@
+package io.nexusrpc;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.jspecify.annotations.Nullable;
+
+/** Information about an operation failure. */
+public class OperationFailure {
```

Comment:
This isn't just for operation failures, we use this type in a few other places in the Go SDK.

Reply:
:+1: Will look into a name change. Hoping to avoid the generic `Failure` class name (Go is not like Java where it will always be qualified in use), but will see if I can find something.

Reply:
Changed to `FailureInfo`